extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcServerProcessor.java

```diff
@@ -66,9 +69,11 @@ MinNettyAllocatorMaxOrderBuildItem setMinimalNettyMaxOrderSize() {
 
     @BuildStep
     void discoverBindableServices(BuildProducer<BindableServiceBuildItem> bindables,
-            CombinedIndexBuildItem combinedIndexBuildItem) {
+            CombinedIndexBuildItem combinedIndexBuildItem,
+            BuildProducer<AnnotationsTransformerBuildItem> annotationTransformers) {
```

Comment:
The `BuildProducer<AnnotationsTransformerBuildItem>` is not used anymore..

Reply:
fixed